lib/dal/src/func/binding.rs

```diff
@@ -305,8 +306,8 @@ impl From<FuncBinding> for si_frontend_types::FuncBinding {
             FuncBinding::Attribute(attribute) => si_frontend_types::FuncBinding::Attribute {
                 func_id: Some(attribute.func_id),
                 attribute_prototype_id: Some(attribute.attribute_prototype_id),
-                component_id: attribute.eventual_parent.into(),
-                schema_variant_id: attribute.eventual_parent.into(),
+                component_id: attribute.eventual_parent.clone().into(),
```

Comment:
Making the `From` use `&` (or maybe better now, adding a `fn component_id(&self) -> Option<ComponentId>`) would avoid cloning (and allocating) here.